tests/integration-tests.sh

```diff
@@ -11,7 +11,7 @@ function stop_docker()
 # Ensure current path is project root
 cd "$(dirname "$0")/../"
 
-QDRANT_VERSION='v0.9.0'
+QDRANT_VERSION='v0.9.1'
```

Comment:
Do we need update version here to `0.10.0` and merge this PR after release?

Reply:
I was planning on updating this PR once the new Docker image is out.

Reply:
Okay, thanks for response!